README.md

```diff
@@ -13,6 +13,7 @@ This repo consists of multiple crates:
 
 | Library | Status | Description |
 |---------|--------|-------------|
+| **Proto** | [![](http://meritbadge.herokuapp.com/trust-dns-proto)](https://crates.io/crates/trust-dns-proto) [![trust-dns-proto](https://docs.rs/trust-dns-proto/badge.svg)](https://docs.rs/trust-dns-proto) | Raw DNS library, containing only the pieces necessary for supporting the other Trust-DNS libraries |
```

Comment:
Maybe expand this to say that the interface -proto exposes is unstable and only for use by the other Trust-DNS libraries, not for end-user use.